doc/contribute/bin_blobs.rst

```diff
@@ -224,5 +224,30 @@ the integration of such blob for the foreseeable future.
 Submission and review process
 =============================
 
-This section will be updated once the standard format for describing binary
-blobs is agreed upon.
+For references to binary blobs to be included in the project, they must be
+reviewed and accepted by the Technical Steering Committee (TSC).
+
+A request for integration with binary blobs must be made by creating a new
+issue in the Zephyr project issue tracking system on GitHub with details
+about the blobs and the functionality they provide to the project.
+
+Follow the steps below to begin the submission process:
+
+#. Make sure to read through the :ref:`bin-blobs` section in
+   detail, so that you are informed of the criteria used by the TSC in order to
+   approve or reject a request
+#. Use the :github:`New Binary Blobs Issue
+   <new?assignees=&labels=RFC&template=bin-blobs.md&title=>` to open an issue
+#. Fill out all required sections, making sure you provide enough detail for the
+   TSC to assess the merit of the request. Additionally you must also create a Pull
+   Request that demonstrates the integration of the binary blobs and then
+   link to it from the issue
+#. Wait for feedback from the TSC, respond to any additional questions added as
+   GitHub issue comments
+
+If, after consideration by the TSC, the conclusion is that integrating external
+source code is the best solution, and the external source code is licensed under
+the Apache-2.0 license, the submission process is complete and the external
+source code can be integrated.
+
+#. On approval of the Zephyr TSC the submission process is complete
```

Comment:
This doesn't look right. "Source code"? And what's up with the indentation?

Reply:
Yep, sorry, will fix.

Reply:
The indentation still looks off. Either that or the last `#. On approval [...]` is redundant, right?